j2lint/linter/rule.py

```diff
@@ -80,20 +82,16 @@ def checkfulltext(self, file, text):
         """
         errors = []
 
-        if not self.checktext:
-            logger.debug("Check text rule does not exist for {}".format(
-                __class__.__name__))
-            return errors
-
         if not self.is_valid_language(file):
             logger.debug(
-                "Skipping file {}. Linter does not support linting this file type".format(file))
+                "Skipping file %s. Linter does not support linting this file type", file)
             return errors
 
         results = self.checktext(file, text)
```

Comment:
So now the assumption is that if a child class does not implement checktext we just use this class `checktext` method and return None? - This works but maybe then state it clearly in the checktext doc string.

Even better could be to return the empty list rather than None

Reply:
if we return an empty list in this , will get another warning for "arguments file and text are not used" that's why returned None.